code/modules/clothing/head/misc.dm

```diff
@@ -181,7 +181,7 @@
 	if(!ishuman(user))
 		return
 	var/mob/living/carbon/human/H = user
-	if(H.get_item_by_slot(ITEM_SLOT_HEAD) == src)
+	if(H.get_item_by_slot(ITEM_SLOT_HEAD) == src && !QDELETED(src)) //This can be called as a part of destroy
```

Comment:
`dropped` is being called by DESTROY? What the FUCK

Reply:
mob del -> contents del -> contents moved off mob -> dropped iirc

Reply:
absurd

Reply:
think of the side effects thatr are happening

Reply:
I cry myself to sleep every night